CHANGELOG.rst

```diff
@@ -5,6 +5,10 @@ Development Version (unreleased):
 
 - Fix unintentional changes in test suite (:pull:`277`).
   By `Kai Mühlbauer <https://github.com/kmuehlbauer>`_
+- Support pyfive as an alternative backend ({pull}`273`).
+  By `Bryan Lawrence <https://github.com/bnlawrence>`_, `David Hassell
+  <https://github.com/davidhassell>`_, and `Valeriu Predoi
+  <https://github.com/valeriupredoi>`_
```

Comment:
Just a note -- the next version of h5netcdf that brings pyfive support should definitely be a major release!